src/app/components/Button/CustomButton/CustomButton.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react'
+import './CustomButton.styles.css'
+import { Button } from 'antd'
+import PropTypes from 'prop-types'
+// import { useTranslation } from 'react-i18next'
+
+function CustomButton(props) {
+  const { buttonType, buttonText, layout = <></>, children } = props
+  // const { ADDITIONAL_DESTRUCTURING_HERE } = user
+
+  // [ADDITIONAL HOOKS]
+  // const { t } = useTranslation('translation')
+  // const { currentLanguage } = t
+
+  // [COMPONENT STATE HOOKS]
+  // const [state, setState] = useState({})
+
+  // [COMPUTED PROPERTIES]
+
+  const style = (buttonType === 'secondary' && 'secondaryButton') || ''
```

Comment:
why you use this syntax, not ternary operator?

Reply:
Idk, works same as ternary operator, it's a habit.